src/transform/index.ts

```diff
@@ -67,8 +67,8 @@ export const transform: PluginImpl<TransformOptions> = () => {
         exports: "named",
         compact: false,
         freeze: true,
-        interop: false,
-        namespaceToStringTag: false,
+        interop: 'esModule',
+        generatedCode: Object.assign({ symbols: false }, options.generatedCode),
```

Comment:
I believe you can use object spread for this.

Reply:
Indeed, it was rather an annoying typescript issue i had to use alternative syntax since it nags about not spreading possibly undefined `generatedCode` (while technically `{ ...undefined}` works in the wonderful land of js!). Happy to do another method if prefer.

Reply:
`{ symbols: false, ...(options.generatedCode || {})` is not working?

Reply:
IMHO, if I'd to choose, I'll prefer `Object.assign({ symbols: false }, options.generatedCode) ` over `{ symbols: false, ...(options.generatedCode || {})}`.